extensions/oidc/deployment/src/main/java/io/quarkus/oidc/deployment/devservices/keycloak/KeycloakDevServicesProcessor.java

```diff
@@ -63,8 +63,8 @@
 
     private static final int KEYCLOAK_EXPOSED_PORT = 8080;
     private static final String KEYCLOAK_DOCKER_REALM_PATH = "/tmp/realm.json";
-    private static final String KEYCLOAK_USER_PROP = "KEYCLOAK_USER";
-    private static final String KEYCLOAK_PASSWORD_PROP = "KEYCLOAK_PASSWORD";
+    private static final String KEYCLOAK_ADMIN_PROP = "KEYCLOAK_ADMIN";
```

Comment:
Will this cause problems with the WildFly version?

Reply:
@stuartwdouglas With WildFly it is `KEYCLOAK_USER=admin KEYCLOAK_PASSWORD=admin`, with this version it is `KEYCLOAK_ADMIN=admin KEYCLOAK_ADMIN_PASSWORD=admin` which are recognized

Reply:
@stuartwdouglas Sorry, yes, it will (not so much these 2 properties - one can create the admin name/password from the KC UI if none was created yet - but dropping of the `/auth` segment will definitely affect it) - but as mentioned in a PR description - it is not meant to be a final change, I've only opened this draft PR to show what needs to be changed to support `-X`.
I'm thinking of adding a property such as `image-distribution` with 2 env values (`quarkus` - default, `wildfly`) and then use the properties/addresses recognized by a given distro. (checking for `-x` in the image name is not very reliable - ex, the custom image name referred to in the linked issue only has `x` but not `-x`)